### ISSUE TITLE
GET /v2/Documents/{documentId} not returning XML format when specified in params

### ISSUE THREAD

**Issue by billdami** [Psychoanalytic-Electronic-Publishing/OpenPubArchive-Content-Server#61]
From what I'm seeing on stage, it doesnt look like this endpoint returns the document in XML format when I use the `return_format=XML` param as specified in the swagger docs, for example this request on stage looks like its still returning HTML format in the `document` field:

```
GET https://stage-api.pep-web.rocks/v2/Documents/Document/APA.027S.0003A/?return_format=XML
```
I also tried the `formatrequested` param that is on the /Search endpoint (which appears to work for search requests returned abstracts data), but didnt seem to have any effect.

**Comment by nrshapiro** [comment]
If you are not authorized, you are getting an abstract in the document field, and it's always in HTML.  The reason is that abstracts are not always provided...sometimes they're generated, and pre-stored in the art_excerpt field.  I decided not to support XML for abstracts because it just doubles what I have to store.  

On your part 2 "try", again Abstracts are never returned in XML in any case.

If you are authorized, you will get XML in the document field, I just tested stage with Postman.

**Comment by nrshapiro** [comment]
@billdami 

Scratch...that...not sure what just happened, but it's not xml...checking...

**Comment by nrshapiro** [comment]
@billdami 

Ok, this is fixed in the 2020.09.29.1 build on stage--I had to pass the return_format quite deep and it wasn't being passed in at the lowest level, so the default html was being returned.  (And I remembered wrong--I guess I put the XML excerpts back in at some point.  I see them in the database, so you can get the abstract back in XML.)  Sorry about the confusion.

**Comment by billdami** [comment]
sounds good @nrshapiro , thanks!
